docs/keys_and_certificates.md

```diff
@@ -221,6 +210,17 @@ keyUsage               = critical, keyCertSign, cRLSign
 subjectKeyIdentifier   = hash
 authorityKeyIdentifier = keyid:always
 ```
+Feel free to adjust configuration to your needs.
+
+* Run command:
+    ```shell
+    openssl req -newkey rsa:2048 -nodes -keyout root_ca_priv.key -x509 -days 365 \
+            -out root_ca.crt -config root_ca.cfg
+    ```
+    - `root_ca_priv.key` - newly created private root CA key. Keep it safe.
+    - `root_ca.crt` - root CA certificate. This will be hardcoded and marked as
+      trusted by Fobnail Token and Attester.
```

Comment:
Can you drop `Attester` while we're at it? We haven't found any use for that.

Reply:
fixed https://github.com/fobnail/docs/pull/49/commits/94d81b369439ddb7020cf55ebe14d7a65a8564d4